app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,10 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_include_tag "application", "data-turbo-track": "reload", type: "module" %>
+
+    <noscript>
+      <meta http-equiv="refresh" content="2;url=nojavascript.html">
```

Comment:
small thing - this doesn't work if the link is nested (for e.g. localhost:3000/zones/2/buildings). Adding '/' in the beginning fixes it:
```
<meta http-equiv="refresh" content="2;url=/nojavascript.html">
```

Reply:
good catch - fixed here [9c09050](https://github.com/lsa-mis/room_ready/pull/75/commits/9c09050a93334697e55a970dfadb959391149d00)